tests/php/ORM/Utf8/Utf8TestHelper.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SilverStripe\Tests\ORM\Utf8;
+
+use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\ORM\DB;
+
+class Utf8TestHelper implements TestOnly
+{
+    use Injectable;
+
+    private string $dbVersion = '';
+
+    public function getUpdatedUtfCharset(string $charset): string
```

Comment:
This method name doesn't really speak to what's happening here imo - it's not clear what "updated" really means in this context. I've added a suggested alternative name but adding a PHPDoc comment for the method would also work if you want to keep the current name.
```suggestion
    public function getExpectedUtfCharsetForCurrentDB(string $charset): string
```

Reply:
Updated